content/blog/astradb-haystack-integration/index.md

```diff
@@ -76,13 +73,10 @@ logging.basicConfig(level=logging.INFO)
 
 embedding_model_name = "sentence-transformers/all-MiniLM-L6-v2"
 
+# Make sure ASTRA_DB_API_ENDPOINT and ASTRA_DB_APPLICATION_TOKEN environment variables are set before proceeding
+
 # embedding_dim is the number of dimensions the embedding model supports.
 document_store = AstraDocumentStore(
```

Comment:
so when I try to run this code, it throws an error:
```
TypeError: AstraDocumentStore.__init__() got an unexpected keyword argument 'embedding_dim'

```
but if I remove the `embedding_dim`, I get a mismatched vector dimension error when I try to run the index pipeline since we aren't supplying the dimensions. Seems like a bug to me?

Reply:
Yep that was an overlook from my side, can you try again?

Reply:
I'm still getting a dimension mismatch error:
```

APIRequestError: {"status":{"insertedIds":[]},"errors":[{"message":"Mismatched vector dimension","errorCode":"INVALID_REQUEST"},{"message":"Mismatched vector dimension","errorCode":"INVALID_REQUEST"},{"message":"Mismatched vector dimension","errorCode":"INVALID_REQUEST"}]}
```

Reply:
actually - I blew away my old database instance and created a fresh one and now the code examples above work. Cool! If you update the credentials list at the top of the blog post, this'll be ready to merge.

thanks so much for updating this, and for your patience!